static/scripts/install-nim-bundle.sh

```diff
@@ -519,6 +532,24 @@ printUsageInfo(){
   exit 0
 }
 
+printSupportedOS(){
+  echo "This script can be run on the following operating systems
+       [-d distribution (ubuntu20.04,ubuntu22.04,ubuntu24.04,debian11,debian12,centos8,rhel8,rhel9,oracle7,oracle8,amzn2)] [-h print help]"
+  printf "\n  ubuntu20.04(focal)"
+  printf "\n  ubuntu22.04(jammy)"
+  printf "\n  ubuntu24.04(noble)"
```

Comment:
24.04 isn't in the tech spec guide -- should it be added?

https://docs.nginx.com/nginx-instance-manager/fundamentals/tech-specs/#supported-distributions

Reply:
I think so, created #145 from your comment.